lib/src/components/AbstractNxPageHeader/AbstractNxPageHeader.tsx

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+import classnames from 'classnames';
+
+import { HeaderLinkProps, Props, propTypes } from './types';
+
+export { Props };
+
+function HeaderLink({ name, href, current }: HeaderLinkProps) {
+  const classes = classnames('nx-page-header__link', {
+    'nx-page-header__link--current': current
+  });
+
+  return (
+    <a className={classes} href={href} data-text={name}>{name}</a>
+  );
+}
+
+// visible for testing
+export { HeaderLinkProps, HeaderLink };
+
+export default function NxPageHeader({ className, links, homeLink, productInfoContent, logo, children }: Props) {
+  const linkEls = links && links.map(link => <HeaderLink key={link.name} { ...link } />),
+      classes = classnames('nx-page-header', className);
+
+  return (
+    <header className={classes}>
+      <div className="nx-page-header__inner">
+        <div className="nx-product">
+          <div className="nx-product__logo">
+            { homeLink ? <a className="nx-product__home-link" href={homeLink}>{logo}</a> : logo }
```

Comment:
I feel like this `<a>` needs either a title or a full-blown tooltip for accessibility.  Right now there's no indication of where it links to.

Reply:
> I feel like this `<a>` needs either a title or a full-blown tooltip for accessibility. Right now there's no indication of where it links to.

I looked around and the sites that are accessibility compliant use `aria-label` on their home links so I have done likewise here: https://github.com/sonatype/sonatype-react-shared-components/pull/189/commits/1bb6949524e49a9966196187b61adb0df4e7f701

I also added it to our main navigation links as well.

Reply:
> I also added it to our main navigation links as well.

-1 on this part.  It is redundant there.

Reply:
> > I also added it to our main navigation links as well.
> 
> -1 on this part. It is redundant there.

I did a bit more reading and I see what you mean, removed: https://github.com/sonatype/sonatype-react-shared-components/pull/189/commits/5197749edb244c873ffaab7d46c7b7e8e6744171